packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -231,7 +231,7 @@ describeCompat("blobs", "FullCompat", (getTestObjectProvider, apis) => {
 		]);
 	});
 
-	[false, true].forEach((enableGroupedBatching) => {
+	[true].forEach((enableGroupedBatching) => {
```

Comment:
We should revisit the test matrix here - may want to ensure coverage with grouping and compression both disabled?